### DISCUSSION TITLE
Is there a way to find out which enhancer regions associated with the TFs that were kept in the final results?

### DISCUSSION THREAD

**Discussion by YaoLi3** [aertslab/scenicplus#504] [Q&A] [answered]
Dear team and fellow Scenic+ users,

I'm wondering if is there a way to find out which enhancer regions associated with the TFs that were kept in the final results?

If I understand correctly, in Scenic+ workflow there is a step where enhancers (start:end) were linked to TFs? If so, where could I find that information, was it stored in any output data file?

Any help will be appreciated!

Best,
Yao

**Comment by SeppeDeWinter** [comment, marked as answer]
Hi @YaoLi3 

Yes, that's stored in the output.

It's stored in the generated `mudata` object `scplus_mdata.uns["direct_e_regulon_metadata"]` and `scplus_mdata.uns["extended_e_regulon_metadata"]` as a pandas dataframe that looks like this

```python


Region | Gene | importance_R2G | rho_R2G | importance_x_rho | importance_x_abs_rho | TF | is_extended | eRegulon_name | Gene_signature_name | Region_signature_name | importance_TF2G | regulation | rho_TF2G | triplet_rank

```

The `Region` column contains the inferred enhancers.

All the best,

Seppe

**Reply by YaoLi3** [reply]
That's great to hear! Thanks a lot for your swift reply. 

However, my results were generated by the older version of Scenic+. Is this as same as `scplus_obj.uns['eRegulon_metadata']`?

Thanks,

Yao

**Reply by SeppeDeWinter** [reply]
Hi @YaoLi3 

Yes, it's the same as `scplus_obj.uns['eRegulon_metadata']`

Best,

Seppe

**Reply by YaoLi3** [reply]
wonderful, thanks again for you kind help
